src/apix-components/apix-api.service.ts

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Inject, Injectable } from '@angular/core';
+import { Http, Headers, RequestOptions, Response } from '@angular/http';
+import { Router } from '@angular/router';
+
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/toPromise';
+import * as _ from 'lodash';
+
+import { ApixUtils } from './apix.utils';
+import { Auth } from './apix.model';
+import { config } from './apix.config';
+
+@Injectable()
+export class ApixApiService {
+
+    public USER_KEY: string = 'apix-user';
+    public SESSION_KEY: string = 'apix-session-id';
+    public SSO_KEY = "apix-sso";
+    public CONFIG_KEY = "apix-config";
+
+    private baseUrl;
+    private remoteApiUrl;
+    private localApiUrl;
+    private remoteSXApiUrl;
+
+    private apixHostLocal = "http://localhost:4200";
```

Comment:
what is this for?

Reply:
Nice catch! The "apixHostLocal" is not used.  I have removed it.